compiler/control/OMROptions.hpp

```diff
@@ -1050,14 +1050,6 @@ enum TR_CompilationOptions
    TR_TraceLRAResults                   = 0x00000800,
    // Available                         = 0x00001000,
 
-   // Register ITF tracing option word
-   //
-   TR_TraceRegisterITFBasic             = TR_TraceGRABasic,
-   // Avaialble                         = 0x00008000,
-   TR_TraceRegisterITFBuild             = 0x00010000,
-   // Available                         = 0x00020000,
-   TR_TraceRegisterITFColour            = 0x00040000,
```

Comment:
The first of these fields (`TR_TraceRegisterITFBasic`) can be safely removed. However, subsequent ones should stay and be commented out as `// Available`. This allows us to keep track of which bits are available for future options.

```c++
   // Avaialble                         = 0x00008000,
   // Avaialble                         = 0x00010000,
   // Available                         = 0x00020000,
   // Avaialble                         = 0x00040000,
```